### ISSUE TITLE
pika URLError Name or service not known

### ISSUE THREAD

**Issue by idallen** [JohnTheNerd/PySarra#8]
For many days now, PySarra is failing:
~~~
2022-12-06 07:05:08,993 - ERROR - pika.adapters.utils.io_services_utils - _consume: protocol.data_received() failed: error=URLError(gaierror(-2, 'Name or service not known')); <ssl.SSLSocket fd=8, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.9.250', 58404), raddr=('205.189.10.47', 5671)>
Traceback (most recent call last):
~~~
I can use `wget` to fetch the hourly file I'm asking for, so that works, but **PySarra** no longer gives me notifications.

[Full error traceback: errs.txt](https://github.com/JohnTheNerd/PySarra/files/10166522/errs.txt)

**Comment by idallen** [comment]
I found the problem.  I had to change this in `main.py`:

    'message': body.split()[1] + body.split()[2]

to this

    'message': body.split()[1] + b'/' + body.split()[2]

**Comment by JohnTheNerd** [comment]
fixed by #10
